### PR TITLE
Fix installation on php 7

### PR DESCRIPTION
**Pull request by rennis23**
https://github.com/Behat/Behat/issues/812

Symfony polyfill component resolve problem with installation and dependency.
https://github.com/symfony/polyfill

### CODE REVIEW

**Comment by jakzal** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -24,7 +23,8 @@
         "symfony/event-dispatcher":      "~2.1|~3.0",
         "symfony/translation":           "~2.3|~3.0",
         "symfony/yaml":                  "~2.1|~3.0",
-        "symfony/class-loader":          "~2.1|~3.0"
+        "symfony/class-loader":          "~2.1|~3.0",
+        "symfony/polyfill":              "^1.0"
```
Comment (stof) [inline composer.json]: please don't depend on the whole polyfill byt only the deps we need

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPENDENCIES='low'
     - php: 5.6
       env: DEPENDENCIES='dev'
+    - php: 7.0
+      env: DEPENDENCIES='dev'
```
Comment (stof) [inline .travis.yml]: no need to test dev dependencies on both 5.6 and 7.0 IMO. Please remove this extra job

**Comment by rennis23** [comment]
This pull request is not necessary.
I closed [issue](https://github.com/Behat/Behat/issues/812), because I had wrong docker configuration (described in issue).

**Comment by ciaranmcnulty** [comment]
The symfony polyfill may still be a good idea!

**Comment by stof** [comment]
@ciaranmcnulty the right solution might be to make the Symfony polyfill provide ``ext-mbstring``, so that it can fulfill such requirement, instead of forcing all Behat users to install it. This would make it opt-in.

**Pull request closed without merging**